public/app/plugins/datasource/loki/components/monaco-query-field/MonacoFieldWrapper.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
@gabor I see we have https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Monaco/ReactMonacoEditorLazy.tsx , so maybe we don't need to do lazy loading here and in Prometheus?

Reply:
@matyax probably yes.
i doublechecked with the browser-dev-tools the current flow:
1. i created a dashboard, added a loki datasource with a query, saved it
2. went back to the dashboard-webpage
3. opened the browser-devtools to see all files loaded
4. did a full reload with cache disabled
5. in the files-that-were-downloaded, i can see the `lokiPlugin` (`2.4MB`). that is ok, we need the loki datasource to display the dashboard. but no monaco, because we do not need the loki-query-field
6. i went into edit-mode for the dashboard-panel.
7. now we loaded `loki-query-field` (`1.2MB`), and monaco's `editor.main.js` (`3.1MB`)

if we remove the lazy-thing, i assume the `loki-query-field` will get bundled with `lokiPlugin`, but maybe they have some shared code so the two of them together might occupy less space.

in other words, i agree, we should remove that lazy-load, but please check the file-sizes afterwards, to be sure everything is OK. the most important thing is that the `3MB` monaco-javascript is not loaded together with `lokiPlugin`. 👍